fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.17.1'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: 'release/0.17.2'
```

Comment:
🚨 Just as a reminder that, as you stated in the PR description, we should not forget to update this back to tag before merging this PR.

Reply:
Done! Updated to `0.18.0`